mxfusion/components/functions/operators/operator_impl.py

```diff
@@ -83,3 +89,43 @@ def reshape(F, data, shape, reverse=False):
 def transpose(F, data, axes=None):
     axes = axes if axes is not None else []
     return F.transpose(data=data, axes=axes)
+
+
+"""Special Operators"""
+
+
+def broadcast_to(data, shape):
+    """The broadcast_to operator"""
```

Comment:
Can you expand this operator's docstring a bit? 
Maybe even include a small example/typical usage like in the docs:
```python
m.x.set_prior(Gaussian(mean=broadcast_to(array([0]), m.x.shape),
                       variance=broadcast_to(array([1]), m.x.shape)))
```